### PR TITLE
Feature/kriii/enhance dockerize

### PR DESCRIPTION
**Pull request by kriii**
- Add json rpc urls (tracker, peer candidate) to start_node_docker.sh
- Add mount a volume readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +170,13 @@ You can use some environment variables, and these have the following options.
 -e ACCOUNT_INJECTION_OPTION={private_key|keystore|mnemonic}
 -e SYNC_MODE={fast|full|peer}
 -e STAKE=<YOUR_TARGET_STAKE>
+-e HOSTING_ENV={gcp|aws}
```
Comment (platfowner) [inline README.md:173]: Could you add 'local' and 'comcom'?

File: README.md [hunk 2]
```diff
@@ -170,6 +170,13 @@ You can use some environment variables, and these have the following options.
...
+```
+You can mount a volume when you meet some wants.
+1. Want to preserve blockchain data in the docker container when changing the docker image due to an update.
+2. Want to save your keystore file in the docker container when injecting your account into the blockchain automatically.
+```
+-v <YOUR_VOLUME>:/home
```
Comment (platfowner) [inline README.md:179]: Can this be changed to <YOUR_VOLUME>:/home/ain_blockchain_data ?

**Comment by kriii** [comment]
PTAL, Thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Nice job done, thanks!

**Comment by kriii** [comment]
Now merged! Thanks for the review!

**Pull request merged**